app/src/main/res/layout/screen_manual_patient_entry.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<org.simple.clinic.newentry.PatientEntryScreen xmlns:android="http://schemas.android.com/apk/res/android"
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Hey why are we not using ConstraintLayout here?

Reply:
The goal of this PR is not to change the root layout to `ConstraintLayout`. We are just migrating the view based screen to a fragment. So there is no need to change the root layout, that can be done later if needed.